core/css.js

```diff
@@ -154,6 +154,7 @@ Blockly.Css.CONTENT = [
     'height: 100%;',
     'position: relative;',
     'overflow: hidden;', /* So blocks in drag surface disappear at edges */
+    'touch-action: none',
```

Comment:
interesting. Should we just add this in develop? or is this somehow required by this gesture stuff?